package-overrides/github/angular/bower-angular-aria@1.4.7.json

```diff
@@ -0,0 +1,12 @@
+{
+  "main": "angular-aria",
+  "shim": {
+    "angular-aria": {
+      "deps": ["angular"]
+    }
+  },
+  "registry": "jspm",
+  "dependencies": {
+    "angular": "^1.4.7"
+  }
+}
```

Comment:
Again, this shouldn't be necessary given the existing override at https://github.com/jspm/registry/blob/master/package-overrides/github/angular/bower-angular-aria%401.3.0.json.